ptcg_cockatrice.xml

```diff
@@ -2487,8 +2485,8 @@ Overload—Reveal a green Pokémon card from your hand.
 			<type>Sorcery</type>
 			<tablerow>3</tablerow>
 			<text>This spell has flash as long as you control a Water Pokémon.
-The owner of target spell and/or Pokémon puts it on the top or bottom of their library.
-Overload—Reveal a blue Pokémon card from your hand.
+The owner of target spell or target Pokémon puts it on the top or bottom of their library.
+Overload {2}{U}{U}, Reveal a blue Pokémon card from your hand.
```

Comment:
If you overload this how does the modal work? you choose "each spell" _or_ "each pokemon", or...?

Reply:
Exactly